docs/_extra/api-reference/schemas/annotation.yaml

```diff
@@ -199,3 +199,24 @@ Annotation:
               description: The annotation creator's display name
               example: "Felicity Nunsun"
             - type: null
+    mentions:
+      type: array
+      items:
+        type: object
+        properties:
+          userid:
+            type: string
+            pattern: "acct:^[A-Za-z0-9._]{3,30}@.*$"
+            description: user account ID in the format `"acct:<username>@<authority>"`
+            example: "acct:felicity_nunsun@hypothes.is"
+          username:
+            type: string
+            description: The username of the user at the time of the mention
+          display_name:
+            type: string
+            description: The display name of the user
+          link:
+            type: string
+            format: uri
+            description: The link to the user profile
+      description: An array of user mentions the annotation text
```

Comment:
Adding here a link to some recent decisions we've made on the schema https://hypothes-is.slack.com/archives/C4K6M7P5E/p1738756919810279

It can be tackled separately on a follow-up PR, or here. Whatever works for you.